src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/commands.py

```diff
@@ -32,6 +32,8 @@
 cli_command(__name__, 'storage account show', 'azure.mgmt.storage.operations.storage_accounts_operations#StorageAccountsOperations.get_properties', factory)
 cli_command(__name__, 'storage account create', 'azure.cli.command_modules.storage.custom#create_storage_account')
 cli_command(__name__, 'storage account list', 'azure.cli.command_modules.storage.custom#list_storage_accounts')
+cli_command(__name__, 'storage account list-account-sas', 'azure.mgmt.storage.operations.storage_accounts_operations#StorageAccountsOperations.list_account_sas', factory)
```

Comment:
Since we now have three commands related to SAS under account (instead of just the one), I would recommend making sas a subgroup so you would have:
`az storage account sas generate` (which should probably be create...)
`az storage account sas list-service`
`az storage account sas list-account`

It would be a breaking change, which is why we should do this prior to GA.